autogen/agentchat/chat.py

```diff
@@ -169,6 +169,10 @@ def initiate_chats(chat_queue: List[Dict[str, Any]]) -> List[ChatResult]:
             - `"carryover"` - It can be used to specify the carryover information to be passed
                to this chat. If provided, we will combine this carryover with the "message" content when
                generating the initial chat message in `generate_init_message`.
+            - `"finished_chat_summary"` - It can be used to specify if past chat summary needs to be passed or not.
+            - `"carryover_indexes"` - It can be used by specifying a list of indexes of the finished_chats list,
+               from which to take the summaries for carryover. If 'carryover_indexes' is not provided or an empty list,
+               then summary from all the finished chats will be taken.
```

Comment:
The docstr here is a little confusing. Are "finished_chat_summary" etc. part of **context? Is "carryover" part of **context? @qingyun-wu could you help explain?

Reply:
Both "carryover" and "finished_chat_summary" shall be part of the input argument dictionary. I will help improve this docstr.

Reply:
Are the extra arguments only valid here and not in `initiate_chat` ?

Reply:
Updated!

Reply:
I think these are only needed here because when one uses 'initiate_chat', there is no native concept of finished chat. If there are, one could manually include them in "carryover".